### PR TITLE
add docker, refactor JsonResponse in account and shop

### PR DESCRIPTION
**Pull request by Vghxv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,18 @@ venv/Scripts/activate
 pip install -r requirements.txt
 python3 main.py
 ```
+## About Database
+```commandline
```
Comment (PEKKA1117) [inline README.md:17]: \`\`\`text  instead

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+FROM python:3.10.11-slim-buster
```
Comment (PEKKA1117) [inline Dockerfile:1]: Remove this file since it's unneeded

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+version: '3.8'
```
Comment (PEKKA1117) [inline docker-compose.yml:1]: Remove this file as it's not needed

**Review by Dragonfly911117 — CHANGES_REQUESTED** [review]

**Comment by Dragonfly911117** [comment]
#49 and #48 have done the docker things, closing it.

**Pull request closed without merging**